wallet/src/main/java/ch/admin/bag/covidcertificate/wallet/detail/CertificateDetailFragment.kt

```diff
@@ -121,6 +123,14 @@ class CertificateDetailFragment : Fragment() {
 		}
 	}
 
+	private fun updateToolbarTitle() {
+		val vaccinationEntry: VaccinationEntry? = dccHolder.euDGC.vaccinations?.firstOrNull()
+		binding.certificateDetailToolbar.setTitle(R.string.covid_certificate_title)
```

Comment:
Might this cause a flickering if we're setting a default title and then change it immediately?